src/server/server_family_test.cc

```diff
@@ -545,4 +545,33 @@ TEST_F(ServerFamilyTest, ConfigNormalization) {
               RespArray(ElementsAre("replica_priority", "13")));
 }
 
+TEST_F(ServerFamilyTest, FlushSearchIndices) {
```

Comment:
I think it belongs to search_family_test even though the bug is related to flush

Reply:
Yes, I also thought to which test it would be better to add it